pages/apis/rest_api/pipeline_templates.md

```diff
@@ -0,0 +1,293 @@
+# Pipeline templates API
+
+> 📘 Enterprise feature
+> [Pipeline templates](https://buildkite.com/docs/pipelines/templates) are only available on an Enterprise plan.
+
+The pipeline templates API allows admins to create and manage pipeline templates for an organization.
+Non-admins will only be permitted to read pipeline templates marked as available by organization admins.
```

Comment:
I think we should use code formatting for "available:"

```suggestion
Non-admins will only be permitted to read pipeline templates marked as `available` by organization admins.
```

Is "read" the right description for what `available` allows for non-admins? In the field description it mentions that when `available` is true, non-admins can assign templates to pipelines.

Reply:
In the context of the pipeline templates rest api, non admin users will only be able to perform `#index` and `#show` read actions on templates marked as `available`.
They'll be able to assign available templates to pipelines via the pipelines rest api :)

Reply:
Okay, so the key difference is not being able to create, update, or delete the actual templates. How about:

> Non-admins can only read or assign pipeline templates marked as `available` by organization admins.